milo/apis/init.lua

```diff
@@ -277,22 +276,15 @@ function Milo:learnRecipe()
 		for _,v1 in pairs(results) do
 			for _,v2 in pairs(ingredients) do
 				if v1.name == v2.name and
-					v1.nbtHash == v2.nbtHash and
-					(v1.damage == v2.damage or
-						(v1.maxDamage > 0 and v2.maxDamage > 0 and
-						 v1.damage ~= v2.damage)) then
+					v1.nbt == v2.nbt then
```

Comment:
Won't this still need to check the damage?  For instance, a hammer used to turn an ingot into a plate, but uses durability.